### PR TITLE
4035 update bitlink

### PR DESCRIPTION
**Pull request by ljdatasci**
I tried testing all of the optional request parameters, but some of them require a paid subscription: link, created_at, launchpad_ids, custom_bitlinks, tags, deeplinks.

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4061"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/bitly/actions/update-bitlink/update-bitlink.mjs [hunk 1]
```diff
@@ -0,0 +1,162 @@
+import bitly from "../../bitly.app.mjs";
+import { formatArrayStrings } from "../../common/utils.mjs";
+import constants from "../../common/constants.mjs";
+
+export default {
+  name: "Update Bitlink",
+  version: "0.0.10",
+  key: "update-bitlink",
```
Comment (feyzullah) [inline components/bitly/actions/update-bitlink/update-bitlink.mjs:8]:
```suggestion
  key: "bitly-update-bitlink",
```

**Review by feyzullah — APPROVED** [review]
LGTM! Just changed version number and action key

**Comment by DilanAthukorala** [comment]
Verification completed and some optional params could not be verified in free version.

**Comment by vunguyenhung** [comment]
Hello @DilanAthukorala, I will merge this PR on behave of the user as they don't have write permission. Thanks!

**Pull request merged**